nitrogen/src/display.rs

```diff
@@ -17,6 +17,27 @@ use crate::resources::semaphore_pool::{SemaphoreList, SemaphorePool};
 use crate::submit_group::ResourceList;
 use std;
 
+pub struct SwapchainSetupConfig {
```

Comment:
Can `#[derive(Copy, Clone, Debug)]` be added here? I am not sure if this will be able to be `Copy`, but `Clone` and `Debug` could be nice :)